src/render_instructions.jl

```diff
@@ -227,7 +227,8 @@ function render_round_rect(
 
     if fstroke
         set_source_rgba(ctx, color_stroke)
-        line_width = abs(user_to_device_distance!(ctx, [line_width,0.0])[1])
+        # from Cairo.user_to_device_distance! - this may be the bug
+        # line_width = abs(user_to_device_distance!(ctx, [line_width,0.0])[1])
         set_line_width(ctx, line_width)
```

Comment:
`Cairo.user_to_device_distance` may being used incorrectly here (`line_width` seems to vary based on heading angle of the vehicle). To be investigated further.

Reply:
Yes it is not clear to me why we need this function. 
I am not sure I understand what it does though: https://www.cairographics.org/manual/cairo-Transformations.html#cairo-user-to-device-distance

Reply:
I think that typically the `line_width` does not scale with the zoom level, so it becomes relatively "thinner" as one zooms in. And the `user_to_device_distance` is used to compensate for that, but maybe there's a bug. I'll open a separate issue so we can take care of it separately.